### ISSUE TITLE
Use reusable method of locating test data

### ISSUE THREAD

**Issue by jayvdb** [coala/coala-bears#1602]
The following bears, and maybe others, all have test data, and have a function to 'find' the test data relative to `__file__`
* CPDBearTest
* DuplicateFileBearTest
* LicenseCheckBearTest
A pytest specific way of doing this is https://pypi.python.org/pypi/pytest-datafiles . However that means other test runners, like [green](https://github.com/coala/coala/issues/4060) would not work.
Simply migrating that algorithm to function in a base test class would suffice ; see https://github.com/coala/coala/issues/3676

**Comment by jayvdb** [comment]
If there isnt an existing re-usable approach, a new one should be added in a standard base class coala/coala#3676

**Comment by Makman2** [comment]
Yes definitely in for standardizing test folders. It's actually not so complicated to find stuff, but it definitely helps and adds one definite way how to handle this. :+1:

**Comment by Makman2** [comment]
For an approach:
I think we can orientate on many tests which have a `load_testdata`-function, which loads a file relative to a directory located in the same directory as the test. So if somebody wants test files:
1. Create a directory with a certain pattern, like `<TestFileName>` or `<TestFileName>TestFiles`
2. Place test-files inside
3. Use `load_testdata`

**Comment by Mixih** [comment]
assigned per GSoC project "Improve Linter Bears"
